modules/blocks.php

```diff
@@ -60,6 +66,50 @@ function jetpack_tiled_gallery_load_block_assets( $attr, $content ) {
 	}
 }
 
+/**
+ * Gif block registration/dependency declaration.
+ *
+ * @param array $attr - Array containing the map block attributes.
+ *
+ * @return string
+ */
+function jetpack_gif_block_render( $attr ) {
+	$align       = isset( $attr['align'] ) ? $attr['align'] : 'center';
+	$padding_top = isset( $attr['paddingTop'] ) ? $attr['paddingTop'] : 0;
```

Comment:
This would indicate we don't know the image size and the element would be either:

- be invisible because it would be inside a 0 height container
- overlap the following content because of its absolute positioning

is that correct? I haven't confirmed which is the case, but either seems to lead to a broken experience. Should we abort rendering the block if this is missing or `0` (falsy)?

Reply:
This is not correct. A fallback size is provided and the iframe is resized to fit within the container.